### PR TITLE
Incorporated Azure Team PR feeback for sdk 

### PR DESCRIPTION
**Pull request by sudhakar4171**

### CODE REVIEW

File: src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/Microsoft.Azure.Management.RecoveryServices.Backup.nuget.proj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.RecoveryServices
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.RecoveryServices.Backup">
-      <PackageVersion>0.1.0</PackageVersion>
+      <PackageVersion>0.1.0-preview</PackageVersion>
```
Comment (dragonfly91) [inline src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/Microsoft.Azure.Management.RecoveryServices.Backup.nuget.proj:8]: Shouldn't it be 1.0.0-preview?

File: src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/RecoveryServicesBackupManagement.csproj [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <ProjectGuid>{14F715EC-C03D-4BB4-83BD-6EEFAA6B0C67}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.Azure.Management.RecoveryServicesBackupManagement</RootNamespace>
+    <RootNamespace>Microsoft.Azure.Management.RecoveryServices.Backup</RootNamespace>
```
Comment (dragonfly91) [inline src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/RecoveryServicesBackupManagement.csproj:9]: We're changing the root namespace? Wouldn't this affect rest of the PS and Portal code?

Comment (sudhakar4171) [inline src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/RecoveryServicesBackupManagement.csproj:9]: no .. this is only for newly added files. Verified both powershell and Portal with new dll and they compiled fine.

File: src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/Microsoft.Azure.Management.RecoveryServices.Backup.nuget.proj (hunk 1 above)
Comment (sudhakar4171) [inline src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/Microsoft.Azure.Management.RecoveryServices.Backup.nuget.proj:8]: PM told to use this version - 0.1.0-preview

**Pull request merged**